src/player.js

```diff
@@ -289,6 +302,14 @@ const MPRISPlayer = new Lang.Class({
         newState.playlist = this._mediaServerPlaylists.ActivePlaylist[1][0];
       }
 
+      let orderings = this._mediaServerPlaylists.Orderings;
+      if (Array.isArray(orderings) && orderings.length > 0) {
+        newState.orderings = orderings;
+      }
+      else {
+        newState.orderings = ['Alphabetical'];
+      }
+
```

Comment:
So here you can also use it:

newState.orderings = getOrderings(this._mediaServerPlaylists.Orderings);